packages/app/src/components/Sidebar/Bookmarks.tsx

```diff
@@ -1,16 +1,80 @@
-import React, { FC } from 'react';
 
+import React from 'react';
+
+import { DevidedPagePath } from '@growi/core';
+import PropTypes from 'prop-types';
 import { useTranslation } from 'react-i18next';
+import { UncontrolledTooltip } from 'reactstrap';
 
-const Bookmarks: FC = () => {
-  const { t } = useTranslation('');
+import LinkedPagePath from '~/models/linked-page-path';
+import { useSWRInifiniteBookmarkedPage } from '~/stores/bookmark';
+import { useCurrentUser } from '~/stores/context';
+
+import PagePathHierarchicalLink from '../PagePathHierarchicalLink';
+
+import InfiniteScroll from './InfiniteScroll';
+
+
+const BookmarksItem = ({ data }) => {
+  const { page } = data;
+  const dPagePath = new DevidedPagePath(page.path, false, true);
+  const linkedPagePathLatter = new LinkedPagePath(dPagePath.latter);
+
+  return (
+    <li className="list-group-item py-3 px-0" id={`bookmark-item-${data._id}`}>
+      <div className="d-flex w-100">
+        <h5 className="my-0">
+          <PagePathHierarchicalLink linkedPagePath={linkedPagePathLatter} basePath={dPagePath.isRoot ? undefined : dPagePath.former} />
+        </h5>
+      </div>
+    </li>
+  );
+
+};
 
-  // TODO Add bookmarks item
+BookmarksItem.propTypes = {
+  data: PropTypes.any,
+};
```

Comment:
- please declare and set type instead using 'propTypes'
-  any is not required...!
(If it's related to bookmarks and not implemented yet, the type should be declare on `interfaces/bookmarks` file)